internal/attribute/attribute_test.go

```diff
@@ -94,3 +94,27 @@ func TestSliceValue(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkBoolSliceValue(b *testing.B) {
+	for n := 0; n < b.N; n++ {
+		_ = BoolSliceValue([]bool{true, false, true, false})
```

Comment:
You are allocating `[]bool{true, false, true, false}` each loop of the benchmark. This should be pre-allocated so it is not include in the benchmark results.